client/src/components/infoDrawer/infoFormat.js

```diff
@@ -0,0 +1,194 @@
+import { H3, H1, UL, Classes } from "@blueprintjs/core";
+import React from "react";
+
+const renderContributors = (contributors, affiliations, skeleton) => {
```

Comment:
nit: Let's use guard clause to rule out the edge case first before dealing with the normal case, this way it's easier to read, because we are not nesting the majority of code when we don't have to.

E.g.,

```ts
const renderContributors = (contributors, affiliations, skeleton) => {
  // eslint-disable-next-line no-constant-condition --  Temp removed contributor section to avoid publishing PII
  if (contributors?.length > 0 && true) return null;

  return (
    <>
      <H3 className={skeleton ? Classes.SKELETON : null}>Contributors</H3>
      <p className={skeleton ? Classes.SKELETON : null}>
        {contributors.map((contributor) => {
          const { email, name, institution } = contributor;

          return (
            <span key={name}>
              {name}
              {email && `(${email})`}
              <sup>{affiliations.indexOf(institution) + 1}</sup>
            </span>
          );
        })}
      </p>
      {renderAffiliations(affiliations, skeleton)}
    </>
  );
};
```